### ISSUE TITLE
Outputting wrong answer on bias function

### ISSUE THREAD

**Issue by joeldmott** [learn-co-curriculum/dsc-bias-variance-trade-off-lab#3]
## Link to Canvas

https://learning.flatironschool.com/courses/5309/assignments/176653?module_item_id=389101

## Issue Subtype

- [ ] Master branch code
- [ ] Solution branch code
- [ ] Code tests
- [ ] Layout/rendering issue
- [ ] Instructions unclear
- [x] Other (explain below)

### Source

```

```

### Concern

Halfway through the lab, we complete a bias function and a variance function. Next, we apply them to our scaled X_train (called "lm_train_predictions" at this point). When I run this cell, my variance output is correct. However, the bias output is not. In fact, it is off by half the correct answer:

![image](https://user-images.githubusercontent.com/51928528/211905034-dd2610d9-b63d-452f-9bf1-435a58e5c671.png)

In the next cell, it asks to do the same thing, but with the test set and this time, both the bias & variance outputs are correct, so something must be going on with the X_train earlier on. 

I went through and double my checked my answers against the solution. I logged into office hours with Morgan Jones, who was very thorough in trying to pinpoint the issue alongside the solution, but after a noble effort, he couldn't spot the problem and recommended I submit via Canvas/GitHub. 

## What OS Are You Using?

- [ ] OS X
- [x] Windows
- [ ] WSL
- [ ] Linux
- [ ] Saturn Cloud from Canvas

## Any Additional Context?

![20230111 Bias-Variance Tradeoff Lab answer discrepancy](https://user-images.githubusercontent.com/51928528/211906125-5f318bfd-9879-4387-995c-4b03ead449a3.jpg)

**Comment by aarongalbraith** [comment]
I had this issue as well, but values in the next cell were also wrong.
<img width="468" alt="image" src="https://user-images.githubusercontent.com/74025302/226447326-ad165440-5224-4456-b9b0-0ef52af36ee8.png">

**Comment by christine-egan42** [comment]
I looked into this issue and here is what I found:

**Error [1](https://github.com/learn-co-curriculum/dsc-bias-variance-trade-off-lab/issues/3#issue-1529612820)**

- The only way that I can get an output that is similar to the output suggested by the comments (**-8.12...e-09**) is to run the code with X_train and not X_train scaled. (see the attached image)
- I changed the suggested answer to the scaled output to avoid confusion in the future to **-4.063...e-09**.

<img width="1240" alt="Screenshot 2023-06-05 at 10 14 10 AM" src="https://github.com/learn-co-curriculum/dsc-bias-variance-trade-off-lab/assets/116017015/83230e47-849e-4b09-adff-91ee481c71dc">

**Error [2](https://github.com/learn-co-curriculum/dsc-bias-variance-trade-off-lab/issues/3#issuecomment-1476827019)**
- To avoid an incorrect answer here, make sure that you are using `.fit_transform()` to apply MinMax scaler to the X_train set, and `.transform()` to X_test.
